### ISSUE TITLE
Inside VueJS template not working?

### ISSUE THREAD

**Issue by robertjoellewis** [mariordev/mailtoui#15]
I tried both the CDN and the require statement using single-page vue templates. adding the class doesn't seem to do anything. No popup. Is there something I am missing here or is this by design?

**Comment by mariordev** [comment]
I just tried it and it's working for me. I wonder if there's something in your setup that's preventing it from working. Could you please share a gist to show me what you have?

**Comment by robertjoellewis** [comment]
Thank you for your quick response! The behavior is that when I click the mailto link, nothing at all happens. However, the demo on your website works just fine. Here is the gist: <script src="https://gist.github.com/robertjoellewis/afbb9f3e6737ffe83f1b89214fc373b1.js"></script>

**Comment by robertjoellewis** [comment]
I also tried the CDN approach, by putting the tag in my index.html file. However, that did not work either.

**Comment by mariordev** [comment]
Everything seems to be ok... This shouldn't really matter, but remove the semicolon at the end of this line: https://gist.github.com/robertjoellewis/afbb9f3e6737ffe83f1b89214fc373b1#file-revealpage-vue-L631

And try this, remove the image inside the `a` tag here: https://gist.github.com/robertjoellewis/afbb9f3e6737ffe83f1b89214fc373b1#file-revealpage-vue-L549

Leave just the text inside that `a` tag and give it a try. Let's see if the image is somehow getting in the way.

**Comment by robertjoellewis** [comment]
That still did not work. However, now I think that it has to do with vue-router. See: https://stackoverflow.com/questions/45387857/router-link-and-href-not-working-in-vue-js

I still can't find a solution yet.

**Comment by mariordev** [comment]
Hmmm... I can see how that could be the issue. However, I don't have a whole lot of experience with vue-router, so it's going to take me a little while to figure it out. I'll see if I can look into it this weekend, but can't promise anything unfortunately.

**Comment by robertjoellewis** [comment]
Of course! The tool is so amazing and I think our project will really benefit from it. I had to temporarily give up for a couple of other deliverables, but I will restart on Monday.
